src/behavior/src/A32NX_Exterior.xml

```diff
@@ -57,28 +57,28 @@
     <Template Name="A32NX_ENGINE_Turbine_Rotation_Template">
         <UseTemplate Name="ASOBO_GT_Anim">
             <!-- 3894 is the max rated N1 of the LEAP, times 6 is from RPM to deg/s, times 0.01 because percentages -->
-            <ANIM_CODE>0.01 6 3894 (A:TURB ENG N1:#ID#, Percent) (A:ANIMATION DELTA TIME, seconds) * * * *</ANIM_CODE>
+            <ANIM_CODE>0.01 6 3856 (L:A32NX_ENGINE_N1:#ID#, Percent) (A:ANIMATION DELTA TIME, seconds) * * * *</ANIM_CODE>
```

Comment:
Which value is correct, the one in the comment or the one in the code? ;)

Reply:
Both values are correct, but there is a reason to change from 3894 to 3856. Nevertheless, the comment will be changed to match the final value.

The maximum rotational speed of the Leap-1A26 is 3894 rpm at a N1 of 101%, but internally the engine model has been adapted to a maximum of 100% (following legacy design considerations). Thus for 100% N1, max  rotational speed would be 3856. This is a purist approach since, for the animation itself, there is no difference whatsoever between 3894 and 3856.

Still, the comment will be updated!

Reply:
Thanks for the explanation!